### ISSUE TITLE
Has 2.1.3 been released for Ubuntu 17.10?

### ISSUE THREAD

**Issue by LordMike** [dotnet/core-setup#4533]
## Steps to reproduce
* Use ubuntu 17.10
* Install dotnet repo as per [this guide](https://www.microsoft.com/net/download/linux-package-manager/ubuntu17-10/runtime-current)
* Check version of various packages

```bash
# apt-cache show dotnet-runtime-2.1
Package: dotnet-runtime-2.1
Version: 2.1.2-1
Description: Microsoft .NET Core Runtime - 2.1.2 Microsoft.NETCore.App 2.1.2
```

```bash
# apt-cache show aspnetcore-runtime-2.1
Package: aspnetcore-runtime-2.1
Version: 2.1.2-1
Description: Microsoft ASP.NET Core 2.1.2 Shared Framework
```

The SDK seems to be (somewhat) up to date.

```
# apt-cache show dotnet-sdk-2.1
Package: dotnet-sdk-2.1
Version: 2.1.302-1
Description: Microsoft .NET Core SDK 2.1.302
```

## Expected  behavior
That `aspnetcore-runtime-2.1` and `dotnet-runtime-2.1` installs the `2.1.3` runtime.

On the downloads page, it _does_ say `2.1.3`:

![image](https://user-images.githubusercontent.com/1027111/45081421-7750e100-b0f7-11e8-8e82-feecb1248f6e.png)

## Actual behavior
That `aspnetcore-runtime-2.1` and `dotnet-runtime-2.1` installs the `2.1.2` runtime.

## Environment data
```
Ubuntu 17.10
```

**Comment by LordMike** [comment]
I have found that on the Ubuntu 18.04 repo, the 2.1.3 packages are available, but it is definitely not on Ubuntu 17.10.

**Comment by leecow** [comment]
Ubuntu 17.10 went EOL prior to the release of .NET Core 2.1.3. 2.1.2 is the last version published for 17.10 as we only support .NET Core on OS versions with also under support by their respective maintainers. 

If you need to run 2.1.3 on 17.10, using the tarball will be the option to try. Download link and instructions can be found on https://github.com/dotnet/core/blob/master/release-notes/2.1/2.1.3/2.1.3-download.md.

**Comment by LordMike** [comment]
Ah - I see .. That clears it up, thanks :)
